src/pages/code-of-conduct.md

```diff
@@ -0,0 +1,40 @@
+---
+layout: "../layouts/PageLayout.astro"
+title: "Code of Conduct"
+---
+
+This code of conduct has been adapted from the [Berlin code of Conduct](https://berlincodeofconduct.org/)
```

Comment:
Does this match the berlin one now or are there still changes? Both are fine with me, but I'd like to know what the changes are if there are any.

Reply:
I changed the first line under "Expected behavior". Initially it said "Participate in an authentic and active way. In doing so, you contribute to the health and longevity of this community.", and I changed it to "- Participate in an authentic and active way. In doing so, you create a positive environment for the conference." since I felt it was more accurate for a conference.

Reply:
But we can change it to say that we just adhere to the Berlin CoC

Reply:
I also removed the section on Open [Culture/Tech/Leadership], since it felt not super appropriate for our conference.

Reply:
Alright! That's all fine with me